resources/js/components/Board.js

```diff
@@ -148,8 +148,8 @@ export class Board extends React.Component {
     }
 
   _onHoldEnd = () => {
-    if (this.props.board_id === 1) {
-      //return;
+    if (this.props.board_id === '1') {
```

Comment:
same here, if you want
```suggestion
    if (this.props.board_id == '1') {
```